plugins/modules/proxmox.py

```diff
@@ -51,6 +52,41 @@
       - See U(https://pve.proxmox.com/wiki/Linux_Container) for a full description.
       - Should not be used in conjunction with O(storage).
     type: str
+  disk_volume:
+    description:
+      - Specify a hash/dictionary of the C(rootfs) disk.
+      - See U(https://pve.proxmox.com/wiki/Linux_Container#pct_mount_points) for a full description.
+    type: dict
```

Comment:
```suggestion
    type: dict
    version_added: 9.2.0
```
Same for the other new option.